src/components/rangeUsePlanPage/versionsList/VersionsToolbar.js

```diff
@@ -43,6 +45,36 @@ const VersionsToolbar = ({
     version: v
   }))
 
+  const StyledMenuItem = withStyles(() => ({
+    root: {
+      width: 500,
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContent: 'flex-start'
+    }
+  }))(MuiMenuItem)
+
+  const ApprovedChip = withStyles(() => ({
+    root: {
+      backgroundColor: '#8bc34a',
+      color: 'white'
+    }
+  }))(Chip)
+
+  const DisabledApprovedChip = withStyles(() => ({
+    root: {
+      backgroundColor: '#9e9e9e',
+      color: 'white'
+    }
+  }))(Chip)
+
+  const createApprovedChip = current =>
+    current ? (
+      <ApprovedChip label="Approved" />
+    ) : (
+      <DisabledApprovedChip label="Approved" disabled />
+    )
+
```

Comment:
Can we move these component definitions outside of the `VersionsToolbar` component? We don't want to unnecessarily re-create these functions on every render.

Reply:
@calebissharp to keep you posted, I ended up removing these component definitions.